packages/git/src/node/dugite-git.ts

```diff
@@ -289,6 +289,8 @@ export namespace GitBlameParser {
 @injectable()
 export class DugiteGit implements Git {
 
+    protected readonly LIMIT = 1000;
```

Comment:
This should be either `protected static LIMIT` or `protected readonly limit`.

Would this worth a dedicated Git preference?